server/src/stm/STM.ts

```diff
@@ -420,6 +420,9 @@ export class CoqStateMachine {
       endCommand();
     }
   }
+  private get proofViewDiff(): boolean {
+    return server.project.settings.coq.proofViewDiff
```

Comment:
I initially thought this should be `this.project` instead of `server.project`, but the current code seems to mix both and hopefully they refer to the same state.

Reply:
It looks like they are the same, but it is probably safer to use `this.project` in case that doesn't continue to be the case.

Reply:
That still worked!

Reply:
The rest looks good to me.